src/site/components/fullwidth_banner_alerts.drupal.liquid

```diff
@@ -41,7 +41,7 @@
             <i aria-label="Close icon" class="fas fa-times-circle"></i>
           </button>
         {% endif %}
-        <div class="usa-alert-body">
+        <div class="usa-alert-body" id="usa-alert-body-text-{{ entity.entityId }}">
           <h3 class="usa-alert-heading">
```

Comment:
Can we move the `id="usa-alert-body-text-{{ entity.entityId }}"` to the H3? That'll be the best way to associate the whole banner with a clear call to action.

Reply:
Yeah, I was shying away from this because there is another method buried in the guts of metalsmith that is adding id's to this (and other) h3's during build. Anywho, I added the id to the h3, add it didn't get wiped out, so.....good to go, @1Copenut :+1: